app/src/main/java/com/orange/ouds/app/ui/utilities/composable/DetailScreenHeader.kt

```diff
@@ -42,16 +44,16 @@ fun DetailScreenHeader(
     @DrawableRes imageRes: Int,
 ) {
     Column {
-        Image(
-            painter = painterResource(imageRes),
-            colorFilter = ColorFilter.tint(OudsColorKeyToken.Content.Default.OnBgEmphasized.value),
-            contentDescription = null,
-            modifier = Modifier
-                .fillMaxWidth()
-                .height(186.dp)
-                .background(OudsColorKeyToken.Background.Emphasized.value),
-            contentScale = ContentScale.None
-        )
+            Image(
```

Comment:
There are additional indent spaces here. `Reformat code` and `Optimize imports` in `Tools/Action on Save` were disabled in Android Studio on my side, maybe it's the same for you.

Reply:
They were enabled but I don't know why sometimes this does not work properly...